docs/credits.md

```diff
@@ -1353,19 +1338,17 @@ Octopus Deploy is made possible thanks to the following great third-party produc
 | path-exists                                       | Sindre Sorhus                                                             | [https://github.com/sindresorhus/path-exists](https://github.com/sindresorhus/path-exists)                                                                                                                                  | [MIT](https://spdx.org/licenses/MIT.html)                                                                           |
 | path-is-absolute                                  | Sindre Sorhus                                                             | [https://github.com/sindresorhus/path-is-absolute](https://github.com/sindresorhus/path-is-absolute)                                                                                                                        | [MIT](https://spdx.org/licenses/MIT.html)                                                                           |
 | path-is-inside                                    | Domenic Denicola                                                          | [https://github.com/domenic/path-is-inside](https://github.com/domenic/path-is-inside)                                                                                                                                      | [WTFPL](https://spdx.org/licenses/WTFPL.html)                                                                      [MIT](https://spdx.org/licenses/MIT.html)                                                                           |
 | path-key                                          | Sindre Sorhus                                                             | [https://github.com/sindresorhus/path-key](https://github.com/sindresorhus/path-key)                                                                                                                                        | [MIT](https://spdx.org/licenses/MIT.html)                                                                           |
 | path-parse                                        | Javier Blanco                                                             | [https://github.com/jbgutierrez/path-parse](https://github.com/jbgutierrez/path-parse)                                                                                                                                      | [MIT](https://spdx.org/licenses/MIT.html)                                                                           |
+| path-scurry                                       | Isaac Z. Schlueter                                                        | [https://github.com/isaacs/path-scurry](https://github.com/isaacs/path-scurry)                                                                                                                                              | [BlueOak-1.0.0](https://spdx.org/licenses/BlueOak-1.0.0.html)                                                       |
```

Comment:
I'm sure this one is OK in principle, however it's not listed as approved here: https://octopushq.atlassian.net/wiki/spaces/RND/pages/3108765730/Using+Open+Source+in+Octopus+projects